server/lib/nl/constants.py

```diff
@@ -337,6 +337,16 @@
     "administrativearea5": "administrative area 5 places",
 }
 
+SCHOOL_TYPE_TO_PLURALS: Dict[str, str] = {
```

Comment:
Any specific reason we shouldn't just add it to PLACE_TYPE_TO_PLURALS?  Please note it as comment.

Reply:
add/combined